services/web/server/src/simcore_service_webserver/folders/_folders_db.py

```diff
@@ -82,12 +88,13 @@ async def create(
             )
             .returning(*_SELECTION_ARGS)
         )
-        row = await result.first()
+        row = result.first()
```

Comment:
I would expect this one to be await maybe the `execute` above should be a `stream`?